packages/components/format-date/src/format-date.stories.ts

```diff
@@ -0,0 +1,160 @@
+import { type Meta, type StoryObj } from '@storybook/web-components';
+import { type TemplateResult, html, nothing } from 'lit';
+import '../register.js';
+import { type FormatDate } from './format-date.js';
+import { formatDateTime } from './format-date-time';
+
+interface Props extends FormatDate {
+  fallback: string | TemplateResult;
+}
+type Story = StoryObj<Props>;
+
+export default {
+  title: 'Utilities/Format date',
+  tags: ['draft'],
+  args: {
+    fallback: 'invalid date',
+    locale: 'en-US',
```

Comment:
If you check the number-field branch, i have an argType for locale as well, with all the locales in use within SL :)

Reply:
I also know that some people don't really like US english, since we're a European company :) I think `'en'` is a better default. Or leave it blank and have it format automatically based on the browser's language.